mula/scheduler/rankers/boefje.py

```diff
@@ -31,11 +31,11 @@ def rank(self, obj: Any) -> int:
         grace_period = timedelta(seconds=self.ctx.config.pq_grace_period)
 
         # New tasks that have not yet run before
-        if obj.prior_tasks is None or not obj.prior_tasks:
+        if obj.latest_task is None or not obj.latest_task:
```

Comment:
What exactly is 'obj' in this scope anyway? Is it the OOI or the to be scheduled job?

Reply:
https://github.com/minvws/nl-kat-coordination/blob/e064fe953c04d93176b4ce7784ff7339b03c2eab/mula/scheduler/schedulers/boefje.py#L566

At this moment a `SimpleNamespace`. I need to review how and what we want to be doing for ranking in the future.